lib/dal/src/func/runner.rs

```diff
@@ -1669,23 +1672,52 @@ impl FuncRunnerExecutionTask {
-                    &self.func,
-                    &serde_json::to_value(args)?,
-                    self.before,
-                )
-                .await
+                // NOTE(nick): changing the behavior is not great because the spans will imply that
+                // a different form of execution ran vs. what actually happened. Why do this here
+                // then? This is the last possible moment before the function executes. Not only
+                // that, but import logic will migrate from the older func to the newer one, so this
+                // will hopefully not be exercised often... and eventually never.
+                match IntrinsicFunc::maybe_from_str(self.func.name.as_str()) {
+                    Some(IntrinsicFunc::ResourcePayloadToValue) => {
+                        info!(
+                            si.func_run.id = %self.func_run.id(),
+                            si.func_run.func.id = %self.func.id,
+                            si.func_run.func.name = %self.func.name,
+                            si.func_run.func.backend_kind = %self.func_run.backend_kind(),
+                            "ignoring JsAttribute func backend kind for ResourcePayloadToValue intrinsic"
+                        );
+                        FuncBackendResourcePayloadToValue::create_and_execute(&self.args).await
+                    }
+                    Some(IntrinsicFunc::NormalizeToArray) => {
+                        info!(
+                            si.func_run.id = %self.func_run.id(),
+                            si.func_run.func.id = %self.func.id,
+                            si.func_run.func.name = %self.func.name,
+                            si.func_run.func.backend_kind = %self.func_run.backend_kind(),
+                            "ignoring JsAttribute func backend kind for NormalizeToArray intrinsic"
+                        );
+                        FuncBackendNormalizeToArray::create_and_execute(&self.args).await
+                    }
+                    Some(_) | None => {
+                        let args = FuncBackendJsAttributeArgs {
+                            component: ResolverFunctionComponent {
+                                data: veritech_client::ComponentView {
+                                    properties: self.args.to_owned(),
+                                    ..Default::default()
+                                },
+                                parents: Vec::new(),
+                            },
+                            response_type: self.func.backend_response_type.try_into()?,
+                        };
+                        FuncBackendJsAttribute::create_and_execute(
+                            self.func_dispatch_context,
+                            &self.func,
+                            &serde_json::to_value(args)?,
+                            self.before,
+                        )
+                        .await
+                    }
```

Comment:
Handles the execution of the old version of these funcs running dynamically with the new intrinsic execution code. We `info!` log this intentionally because you will not see it on the veritech resolver function side now.